docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -462,7 +462,9 @@ All projects can have a `pre` and `post` field which define a list of commands
 to run at the start and end of every task that isn't in a task group. For task
 groups, `setup_task` and `teardown_task` will run instead of `pre` and `post`
 (see [task groups](#task-groups) for more information). These are incredibly
-useful as a place for results commands or for task setup and cleanup.
+useful as a place for results commands or for task setup and cleanup. Note: If a 
+host runs into an issue and needs to exit before it ran the post task commands, 
```

Comment:
nit: maybe we drop "before it ran the post task commands" since this seems redundant of line 467

Reply:
(same below)